Resources/Prototypes/Catalog/Fills/Paper/manuals.yml

```diff
@@ -26,6 +26,8 @@
     netsync: false
     sprite: Objects/Misc/guardian_info.rsi
     state: guardian_info
+  - type: ActivatableUI
+    key: enum.PaperUiKey.Key
```

Comment:
A look at ActivatableUIComponent tells me this is incorrect. https://github.com/space-wizards/space-station-14/blob/108c7cfdf6d1aa24d85ae89bc4da0078910ba4d9/Content.Server/UserInterface/ActivatableUIComponent.cs

Reply:
Otherwise does not work.